javascript/packages/orchestrator/src/configGenerator.ts

```diff
@@ -618,6 +631,9 @@ async function getNodeFromConfig(
     p2pCertHash: node.p2p_cert_hash,
   };
 
+  if (node.prometheus_prefix)
+    nodeSetup.prometheusPrefix = node.prometheus_prefix;
```

Comment:
If ```prometheus_prefix``` is ```some value || undefined``` it will stay ```undefined``` 

```suggestion
nodeSetup.prometheusPrefix = node.prometheus_prefix;
```

Reply:
Not really - prior to this if the key `prometheusPrefix` does not exist; With your suggestion, it will create the key and "assign" `undefined` to it